weather-app/src/styles.scss

```diff
@@ -63,5 +58,5 @@ button {
 .main-container {
   margin: 0 auto;
   padding: 0 20px;
-  width: 1170px;
+  width: 1210px;
```

Comment:
Do we need this change?

Reply:
Yes, we decided to make the content width 1170 + padding 20

Reply:
You changed 1170 to 1210 and after that => 1210 + 40 = 1250
Before we had 1170  => 1170 + 40  = 1210.

Sorry i don't understand